index.html

```diff
@@ -0,0 +1,212 @@
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+    <style>
+      @import url("https://fonts.googleapis.com/css2?family=Lato:wght@300;900&display=swap");
+    </style>
+
+    <link
+      rel="stylesheet"
+      href="https://use.fontawesome.com/releases/v5.14.0/css/all.css"
+      integrity="sha384-HzLeBuhoNPvSl5KYnjx0BT+WB0QEEqLprO+NBkkk5gbc67FTaL7XIGa2w1L0Xbgc"
+      crossorigin="anonymous"
+    />
+
+    <link
+      rel="stylesheet"
+      href="https://use.fontawesome.com/releases/v5.14.0/css/all.css"
+      integrity="sha384-HzLeBuhoNPvSl5KYnjx0BT+WB0QEEqLprO+NBkkk5gbc67FTaL7XIGa2w1L0Xbgc"
+      crossorigin="anonymous"
+    />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+    <style>
+      @import url("https://fonts.googleapis.com/css2?family=Alfa+Slab+One&display=swap");
+    </style>
+    <title>Visit Rwanda Festivals and Events</title>
+  </head>
+  <body>
+    <div class="social_icon">
+      <ul>
+        <li class="fa-size">
+          <a href="#"><i class="fa fa-twitter" aria-hidden="true"></i></a>
+        </li>
+        <li class="fa-size">
+          <a href="#"><i class="fa fa-linkedin" aria-hidden="true"></i></a>
+        </li>
+        <li><a href="#">English</a></li>
+        <li><a href="#">My Page</a></li>
+        <li><a href="#">Logout</a></li>
+      </ul>
+    </div>
+    <header id="header">
+      <div class="logo">
+        <a href="index.html">
+          <img src="images/gorilla-logo.png" alt="logo"
+        /></a>
+      </div>
+      <nav id="nav-bar" class="nav-bar d-none">
+        <div class="nav-wrap">
+          <a class="nav-link" id="about" href="aboutpage.html">About</a>
+          <a class="nav-link visited" href="#programs">Program</a>
+          <a class="nav-link" href="#contents">Join</a>
+          <a class="nav-link" href="#">News</a>
+          <a class="nav-link" href="#sponsor">Sponsor</a>
+          <a class="nav-link" id="home" href="index.html">RW Compaign</a>
+        </div>
+      </nav>
+      <div class="menu">
+        <i class="fas fa-bars"></i>
+      </div>
+      <div id="close-icon" class="close-icon d-none">
+        <i class="fas fa-times-circle"></i>
+      </div>
+    </header>
+    <div class="banner">
+      <div class="background">
+        <h1>"Rwanda Film Festival"</h1>
+        <h2>
+          Visit Rwanda Festivals <br />
+          and Events
+        </h2>
+        <p>
+          If you’re looking to get out of the big city and back into nature, the
+          annual Kwita Izina gorilla-naming ceremony is a special time to visit
+          Volcanoes National Park. The annual naming ceremony takes place in
+          Kinigi,
+        </p>
+        <h3>24th SEPTEMBER 2021 SAVE THE DATE.</h3>
+      </div>
+    </div>
+    <main id="programs">
```

Comment:
This issue is not properly resolved.
- [ ] Your _main program_ section does not follow the typography (**font family**, **font-weight**, ...). Kindly ensure to use the appropriate fonts properties (**font family**, **font-weight**, font size...)). Please bear in mind that we want our design to as much as possible follow the provided design template as much as possible.

**Your program section:**
![image](https://user-images.githubusercontent.com/12745474/133397262-0228642b-c9d8-4dc1-84b6-3c4947f875ec.png)


**The original program section:**
![image](https://user-images.githubusercontent.com/12745474/132545825-150c3b77-7c05-4a8c-aa95-956269677499.png)

Reply:
this should be the browser zoom that might be different, mine is 100% zoom